### ISSUE TITLE
Bessere Lösung für Sprachwechsel finden

### ISSUE THREAD

**Issue by demianh** [bula21/mova21-app#94]
Scheint bei mehreren Personen ein Problem zu sein

**Comment by demianh** [comment]
Eine Möglichkeit wäre: Beim ersten Start fragen

**Comment by jakmeier** [comment]
@demianh Bin mir nicht sicher, was hier wirklich ideal ist. Eine Frage beim ersten Start löst doch nur das Problem, wenn die Default-Auswahl nicht funktioniert. Wir haben aber immer noch das Problem, dass gewisse Leute auf Englisch wechseln werden und dann nicht alles sehen. Und dass wer die Sprache wechseln will, nicht weiss wo man das tut. ("Infos" ist nicht gerade intuitiv.)

Wäre es eventuell nicht doch besser, die Sprachenänderung auf jeder Seite zu ermöglichen? Und dann dafür bei den News oben eine Warnung anzeigen, wenn man auf Englisch eingestellt hat, dass man in anderen Sprachen mehr sehen würde.

**Comment by demianh** [comment]
Ja, beim ersten Start anzeigen würde zumindest das Problem entschärfen, von Personen die Französisch/Italienisch sprechen und ihr Telefon auf Englisch eingestellt haben.

Eine Meldung beim Englisch anzeigen finde ich sinnvoll. Kann man so oder so machen.

Auf jeder Seite kann man die Sprache nicht wechseln, da wir die unterschiedlichen Sprach-Pages nicht verlinkt haben. Man kann also eh nur auf der Übersicht wechseln. Aber wo man es am Besten Platzieren soll, ist die Frage...
